src/config.rs

```diff
@@ -824,7 +824,9 @@ impl Merge for GlobalPassOption {
 
 impl Merge for react::Options {
     fn merge(&mut self, from: &Self) {
-        *self = from.clone();
+        if *from != react::Options::default() {
```

Comment:
@kdy1 I'm not really sure about this, but I noticed when testing via the CLI that the React options in my `.swcrc` file were not being used at all. This appears to be because `from` here are the default options, which overwrite the options from the config. Not sure if the merging order should be different, or perhaps the API should be passed an optional in case no options are given rather than using the defaults. Let me know what you want to do.

Reply:
>  I noticed when testing via the CLI that the React options in my .swcrc file were not being used at all

This seems like a bug. Maybe I need to add some more tests for `.swcrc`.
I think it should respect `.swcrc`, while preferring the option passed via javascript api.

Reply:
I didn't look too far, but it could be here where the defaults are being used in case no options are passed via the API: https://github.com/swc-project/swc/blob/a3105428ba2ff9abd7d0e5464868da3ca9b66294/native/src/transform.rs#L71

Reply:
Yeah, you are right.
But I'm not sure about the fix at the moment.
If you want, you can just leave it and I'll fix it with another pr.

Reply:
Sounds good